lock-impl/src/test/java/com/palantir/lock/logger/LockServiceStateLoggerTest.java

```diff
@@ -117,34 +115,30 @@ public static void setUp() throws Exception {
     public void testFilesExist() throws Exception {
         List<File> files = LockServiceTestUtils.logStateDirFiles();
 
-        assertEquals(
-                "Unexpected number of descriptor files",
-                files.stream()
+        assertThat(1)
```

Comment:
might be good to fix this batch: the `files.stream(...)` is the actual value (and so should be in `assertThat(...)`) while the expected value is `1`.